src/Relation/Binary/Properties/Setoid.agda

```diff
@@ -0,0 +1,29 @@
+------------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Additional properties for setoids
+------------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+open import Relation.Binary
+
+module Relation.Binary.Properties.Setoid {a ℓ} (S : Setoid a ℓ) where
+
+open import Function using (_∘_; _$_)
+open import Relation.Nullary using (¬_)
+
+open Setoid S using (Carrier; _≈_; refl; sym; trans)
+
+_≉_ :  Rel Carrier _
+_≉_ x =  ¬_ ∘ (x ≈_)
```

Comment:
Note that `_≉_` is now defined within `Setoid` so it no longer needs to be defined here.